src/Symfony/Component/Ldap/Adapter/ExtLdap/Collection.php

```diff
@@ -44,31 +44,40 @@ public function toArray()
 
     public function count()
     {
-        if (false !== $count = ldap_count_entries($this->connection->getResource(), $this->search->getResource())) {
-            return $count;
+        $con = $this->connection->getResource();
+        $searches = $this->search->getResources();
+        $count = 0;
+        foreach ($searches as $search) {
```

Comment:
`foreach ($this->search->getResources() as $search) {`

Reply:
Addressed; thanks!